packages/shared/src/components/search/SearchContainer.tsx

```diff
@@ -20,7 +20,7 @@ export function SearchContainer({
   chunk,
 }: SearchContainerProps): React.ReactElement {
   return (
-    <div className="m-auto grid w-full max-w-screen-laptopL grid-cols-1 gap-y-6 py-8 laptop:grid-cols-3">
+    <div className="m-auto grid w-full max-w-screen-laptopL grid-cols-1 gap-y-6 py-8 laptop:grid-cols-3 mt-14 laptop:mt-auto">
```

Comment:
Check mobile because it is separate styling then desktop. 🙏

Reply:
yes have checked, this was done to accommodate mobile view, see images on comments above